internal/controller/services/monitoring/monitoring_controller_support.go

```diff
@@ -6,41 +6,32 @@ import (
 	"errors"
 
 	serviceApi "github.com/opendatahub-io/opendatahub-operator/v2/api/services/v1alpha1"
-	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
 	odhtypes "github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
 )
 
 //go:embed resources
 var resourcesFS embed.FS
 
 const (
-	MonitoringStackTemplate = "resources/monitoring-stack.tmpl.yaml"
-	ManagedStackName        = "rhoai-monitoringstack"
-	OpenDataHubStackName    = "odh-monitoringstack"
+	MonitoringStackTemplate        = "resources/monitoring-stack.tmpl.yaml"
+	OpenTelemetryCollectorTemplate = "resources/opentelemetry-collector.tmpl.yaml"
+	CollectorRBACTemplate          = "resources/collector-rbac.tmpl.yaml"
+	PrometheusRouteTemplate        = "resources/prometheus-route.tmpl.yaml"
+	MonitoringStackName            = "monitoringstack"
+	PrometheusPipelineName         = "odh-prometheus-collector"
```

Comment:
Do we want to have just one name? In that case, we could prefix it with `default-` as we do for other resources?

Reply:
I prefer to keep the name short, because these names will likely end up in documentation - e.g. for scaling.

For the collector pipeline, using a prefix is valuable because it reduces the chance that a customer will add a conflicting pipeline for 3rd party export, but there's no value in having it be different between ODH and RHOAI deployments.